debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
Should we not use the LICENSE from mavlink repo instead?

Reply:
Indeed, I failed to fix the license in this file -- fixed now.